extensions/wikia/Hydralytics/classes/Information.php

```diff
@@ -0,0 +1,321 @@
+<?php
+/**
+ * Curse Inc.
+ * Hydralytics
+ * Information Class
+ *
+ * @author		Alexia E. Smith
```

Comment:
That could be potentially misleading - I'd change ownership information here or mark in our own redshift etc. classes to contact us for these additions

Reply:
Resolved in 5736136b26b